src/theme/book.js

```diff
@@ -329,46 +335,58 @@ if (window.playground_copyable) {
 })();
 
 (function clipboard() {
-	var clipButtons = document.querySelectorAll('.clip-button');
+	const clipButtons = document.querySelectorAll('.clip-button');
 
 	function hideTooltip(elem) {
 		elem.firstChild.innerText = "";
-		elem.className = 'fa fa-copy clip-button';
+		elem.className = 'fa far fa-clipboard clip-button';
 	}
 
 	function showTooltip(elem, msg) {
 		elem.firstChild.innerText = msg;
-		elem.className = 'fa fa-copy tooltipped';
+		elem.className = 'fa far fa-clipboard tooltipped';
 	}
 
-	var clipboardSnippets = new ClipboardJS('.clip-button', {
-		text: function(trigger) {
+	const clipboardSnippets = new ClipboardJS('.clip-button', {
+		text: (trigger) => {
 			hideTooltip(trigger);
-			let playground = trigger.closest("pre");
+			const playground = trigger.closest("pre");
 			return playground.querySelector("code").textContent;
 		}
 	});
 
-	Array.from(clipButtons).forEach(function(clipButton) {
-		clipButton.addEventListener('mouseout', function(e) {
+	clipButtons.forEach((clipButton) => {
+		clipButton.addEventListener('mouseout', (e) => {
 			hideTooltip(e.currentTarget);
 		});
 	});
 
-	clipboardSnippets.on('success', function(e) {
+	clipboardSnippets.on('success', (e) => {
 		e.clearSelection();
 		showTooltip(e.trigger, "Copied!");
 	});
 
-	clipboardSnippets.on('error', function(e) {
+	clipboardSnippets.on('error', (e) => {
 		showTooltip(e.trigger, "Clipboard error!");
 	});
 })();
 
+(function wrap() {
+	const wrapButtons = document.querySelectorAll(".wrap-button");
+	wrapButtons.forEach((button) => {
+		button.addEventListener('click', (e) => {
+			const playground = button.closest("pre");
+			const codeBlock = playground.querySelector("code");
+			if (!codeBlock.style.textWrap || codeBlock.style.textWrap === 'nowrap') codeBlock.style.textWrap = 'wrap';
+			else codeBlock.style.textWrap = 'nowrap';
```

Comment:
Let's keep the code style consistent
```suggestion
			if (!codeBlock.style.textWrap || codeBlock.style.textWrap === 'nowrap') {
			    codeBlock.style.textWrap = 'wrap';
			} else {
			    codeBlock.style.textWrap = 'nowrap';
			}
```